### PR TITLE
Add Prometheus example

### PR DESCRIPTION
**Pull request by avestuk**
Uses Prometheus operator and then guides creation of a StorageOS specific ServiceMonitor. Also  explains how to use Grafana to query Prometheus for metrics.

### CODE REVIEW

File: k8s/examples/prometheus/prometheus-cr.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+apiVersion: monitoring.coreos.com/v1
+kind: Prometheus
+metadata:
+  name: prometheus-storageos
+  labels:
+    app: prometheus-operator
+spec:
+  #serviceMonitorSelector: {} This probably makes it monitor all
```
Comment (Arau) [inline k8s/examples/prometheus/prometheus-cr.yaml]: I don't think this line is meant to be released

**Review by Arau — APPROVED** [review]
Everything runs and follows smoothly.

**Review by alex-leonhardt — APPROVED** [review]
LGTM

File: k8s/examples/prometheus/prometheus-cr.yaml (hunk 1 above)
Comment (avestuk) [reply k8s/examples/prometheus/prometheus-cr.yaml]: Indeed not. I've updated the file.

**Pull request merged**